records_mover/url/base.py

```diff
@@ -151,6 +179,7 @@ def upload_fileobj(self, fileobj: IO[bytes], mode: str = 'wb') -> int:
             return blcopyfileobj(fileobj, local_file)
 
     def download_fileobj(self, output_fileobj: IO[bytes]) -> None:
+        # TODO: document
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker